libs/features/mas/src/merch-card.js

```diff
@@ -365,6 +365,7 @@ export class MerchCard extends LitElement {
     handleQuantitySelection(event) {
         const allLinks = [
             ...this.checkoutLinks,
+            ...this.prices,
```

Comment:
nit: price is not a link, I would rename `allLinks` variable.

Reply:
Renamed to allMerchElements. Unit tests added to template.test.js